.github/workflows/ci-test.yml

```diff
@@ -50,6 +51,11 @@ jobs:
         uses: ./.github/actions/matches
         with:
           pattern: 'Dockerfile$\|.*.sh$'
+      - id: helm-changes
+        name: find helm changes
+        uses: ./.github/actions/matches
+        with:
+          pattern: '^helm'
```

Comment:
I'd also recommend adding in the .github directory as a trigger since the job uses workflows/actions